sdks/java/io/google-cloud-platform/src/test/resources/org/apache/beam/sdk/io/gcp/pubsub/all_data_types_flat_schema.proto

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+syntax = "proto3";
```

Comment:
I feel silly but - does Pubsub not support also JSON schemas?

Reply:
@pabloem Pub/Sub schemas only support Avro and proto.  The Avro schema is configured using JSON.  See https://cloud.google.com/pubsub/docs/schemas#schema_types